packages/zoe/src/contractFacet/types.ts

```diff
@@ -7,6 +9,13 @@ import type {
   Issuer,
   Payment,
 } from '@agoric/ertp';
+import type { Subscriber } from '@agoric/notifier';
```

Comment:
ERef, Pattern, Subscriber and such were somehow ambiently in scope here?

Reply:
yeah there were some ambient declarations pretty deep down in the dependency graph